src/plots.jl

```diff
@@ -0,0 +1,3 @@
+# Stubs are extended in ext/RecipesBaseExt.jl
+macro enable() end
```

Comment:
This means packages need to do `import RecipesBase` before using this macro? Is that the idea?

Reply:
That would work, but like my example, I would expect `using Plots`. Anything that would load RecipesBase will trigger it.

Reply:
And in terms of packages, yes RecipesBase would work, and so would their existing dependency on GeoInterfaceRecipes (since it depends on RecipesBase).